src/components/global/ContactFormModal.jsx

```diff
@@ -132,6 +144,12 @@ const ContactFormModal = ({ showContactFormModal, setShowContactFormModal }) =>
           </Grid>
         </DialogActions>
       </Box>
+      {messageSuccess ? (
```

Comment:
These work just fine, but pretty common in react to use the Javascript short-circuiting with `&&` operator VS a ternary.

Reply:
I'm fine with either, but wanted to add that I came across this article:

https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx

Reply:
Thanks for this. I was aware of issue with possible 0, but not undefined.

In this case the variables/states are either true or false, no other option - so I'd prefer the `&&`. 

In case of array length or something that could end up being falsy somehow, then the explicit ternary is ideal.

It's just a personal preference, I'm more than happy to conform to ternaries if you or others prefer.